rust/gsc/src/main.rs

```diff
@@ -0,0 +1,51 @@
+use axum::{
+    // http::StatusCode,
+    routing::post,
+    Json,
+    Router,
+};
+use entity::job;
+use migration::{Migrator, MigratorTrait};
+use sea_orm::{ActiveModelTrait, ActiveValue, DatabaseConnection};
+use serde::Deserialize;
+use std::sync::Arc;
+
+#[derive(Deserialize)]
+struct AddJobPayload {
+    cmd: String,
+    env: String,
+}
+
+async fn add_job(Json(payload): Json<AddJobPayload>, conn: Arc<DatabaseConnection>) {
+    let insert_job = job::ActiveModel {
+        id: ActiveValue::NotSet,
+        cmd: ActiveValue::Set(payload.cmd),
+        env: ActiveValue::Set(payload.env),
+    };
+    insert_job.insert(conn.as_ref()).await.unwrap();
```

Comment:
We should handle the error here instead of `unwrap()`ing

Reply:
So the only options for error handling here are 1) to return a particular StatusCode (which we should probably do) or 2) to use tracing to log the error (which we should probably *ALSO* do) which is a bit unconventional to the normal `.map_err(...)?` that you might otherwise do.

Reply:
ok added this but the tracing only outputs to stdout right now, eventually we can add an append only log of some kind.